test/tests/functional/pbs_cray_reliable_job_startup.py

```diff
@@ -169,31 +169,28 @@ def test_reliable_job_startup_not_supported_on_cray(self):
         job_stat = self.server.status(JOB, id=jid)
         exechost = job_stat[0]['exec_host'].partition('/')[0]
         mom_superior = self.moms[exechost]
-        found = mom_superior.log_match(
-            msg, starttime=start_time, regexp=True, max_attempts=10)
-        self.assertTrue(found)
+        mom_superior.log_match(msg, starttime=start_time)
 
         # Check that 's' record is absent since release_nodes() was not called
-        try:
-            self.server.accounting_match(
+        self.server.accounting_match(
                 msg=".*%s;%s;.*" % ('s', jid),
-                regexp=True, n=50, max_attempts=3)
-        except PtlLogMatchError:
-            self.logger.info(
+                regexp=True, n=50, max_attempts=3, existence=False)
```

Comment:
I would update max_attempts to 10 to avoid the false positives

Reply:
Done